recsourcePayoutRefactor.jl

```diff
@@ -0,0 +1,83 @@
+using Printf
+
+# Province struct
+struct Province
+    name::String
+    money::Int
+end
+
+# Function to generate the power set of provinces
+function power_set(country::Vector{Province})

```

Comment:
We should be able to use Combinatorics.jl for this.

Reply:
Done.